src/utils/image.ts

```diff
@@ -0,0 +1,8 @@
+type TransformImageOptions = {
+  width: number;
+  height: number;
+  resize?: 'fit' | 'smartcrop';
+};
+
+export const transformImage = (imageSrc: string, { width, height, resize = 'fit' }: TransformImageOptions) =>
```

Comment:
We increase the requested image size by 1.2 factor as Netlify provides compression that lowers its quality. With this transformation, quality loss is less visible.

Reply:
🧙
Can you please add this comment in the code?